tests/pipeline/test_blaze.py

```diff
@@ -1876,6 +1876,54 @@ def test_checkpoints_out_of_bounds(self):
         self._test_checkpoints(checkpoints)
 
 
+class LastInGroupSortingTestCase(BlazeToPipelineTestCase):
```

Comment:
then you can pass your own dates around in the function instead of using self.dates.